ad-fmcjesdadc1-ebz/ad9250.c

```diff
@@ -105,6 +105,10 @@ int32_t ad9250_setup(struct ad9250_dev **device,
 
     /* Setup SPI descriptor */
     ret = spi_init(&dev->spi_dev, &init_param.spi_init);
+	if(ret < 0) {
+		ad_printf("Error spi_init()\n");
```

Comment:
if you just want to print the function name you can use __func__ if the ad_printf accepts parameters like %s

ad_printf("Error %s\n", __func__);

Reply:
In this case the current function is not spi_init but ad9250_setup, but the error is from SPI